bin/live/pycbc_live_supervise_collated_trigger_fits

```diff
@@ -0,0 +1,626 @@
+            stat_files=stat_files,
+        )
+        plot_single_significance_fits(
+            ssf_file,
+            daily_plot_options,
+            controls
+        )
+    if args.combine_significance_fits:
+        csf_file, date_str = combine_significance_fits(
+            combined_options,
+            combined_control_options,
+            date_str,
+            ifo,
+            controls
+        )
+        plot_combined_significance_fits(
+            csf_file,
+            date_str,
+            combined_plot_options,
+            combined_plot_control_options,
+            controls
+        )
+
+
+parser = argparse.ArgumentParser(description=__doc__)
+pycbc.add_common_pycbc_options(parser)
+parser.add_argument(
+    '--config-file',
+    required=True
+)
+parser.add_argument(
+    '--date',
+    help='Date to analyse, if not given, will analyse yesterday (UTC). '
+         'Format YYYY_MM_DD. Do not use if using --run-daily-at.'
+)
+parser.add_argument(
+    '--fit-by-template',
+    action='store_true',
+    help="Perform template fits calculation."
+)
+parser.add_argument(
+    '--fit-over-multiparam',
+    action='store_true',
+    help="Perform template fits smoothing."
+)
+parser.add_argument(
+    '--single-significance-fits',
+    action='store_true',
+    help="Perform daily singles significance fits."
+)
+parser.add_argument(
+    '--combine-significance-fits',
+    action='store_true',
+    help="Do combination of singles significance fits."
+)
+parser.add_argument(
+    '--run-daily-at',
+    metavar='HH:MM:SS',
+    help='Stay running and repeat the fitting daily at the given UTC hour.'
+)
+args = parser.parse_args()
+
+pycbc.init_logging(args.verbose, default_level=1)
+
+if args.run_daily_at is not None and args.date is not None:
+    parser.error('Cannot take --run-daily-at and --date at the same time')
+
+if args.run_daily_at is not None:
+    # keep running and repeat the fitting every day at the given hour
+    if not re.match('[0-9][0-9]:[0-9][0-9]:[0-9][0-9]', args.run_daily_at):
+        parser.error('--run-daily-at takes a UTC time in the format HH:MM:SS')
+    logging.info('Starting in daily run mode')
+    while True:
+        sv.wait_for_utc_time(args.run_daily_at)
+        logging.info('==== Time to update the single fits, waking up ====')
+        # Get the date string for yesterday's triggers
+        day_dt = datetime.utcnow() - timedelta(days=1)
+        day_dt = datetime.combine(day_dt, datetime.min.time())
+        day_str = day_dt.strftime('%Y_%m_%d')
+        supervise_collation_fits_dq(args, day_dt, day_str)
+else:
+    # run just once
+    if args.date:
+        day_str = args.date
+        day_dt = datetime.strptime(args.date, '%Y_%m_%d')
+    else:
+        # Get the date string for yesterday's triggers
+        day_dt = datetime.utcnow() - timedelta(days=1)
+        day_dt = datetime.combine(day_dt, datetime.min.time())
+        day_str = day_dt.strftime('%Y_%m_%d')
```

Comment:
These four lines are repeated identically in the other branch. I suggest moving them to a function `get_yesterday_str()` and using the comment for its docstring.